starter-code/scripts/articleView.js

```diff
@@ -18,17 +18,21 @@ articleView.populateFilters = function() {
 
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-  if ($(this).val()) {
+    if ($(this).val()) {
       /* TODO: If the slect box changes to an option that has a value, we should:
           1. Hide all of the articles
           2. Fade in only the articles that match based on on the author
             that was aselected. Hint: use an attribute selector to find
             those articles that match the value, and then fade them in.
         */
-  } else {
+      $('article').hide();
```

Comment:
This will definitely work, but you should be more specific in your selectors if you can. If you had article elements on your page beyond the ones in the articles section, this would hide those as well. Instead, you could refer to `#articles article` which will point to all articles inside of the section with the #articles ID.